HW_3/task_20.py

```diff
@@ -0,0 +1,25 @@
+# В настольной игре Скрабл (Scrabble) каждая буква имеет определенную ценность. 
+# В случае с английским алфавитом очки распределяются так:
+# A, E, I, O, U, L, N, S, T, R – 1 очко;
+# D, G – 2 очка; 
+# B, C, M, P – 3 очка;
+# F, H, V, W, Y – 4 очка; 
+# K – 5 очков;
+# J, X – 8 очков; 
+# Q, Z – 10 очков. 
+# А русские буквы оцениваются так: А, В, Е, И, Н, О, Р, С, Т – 1 очко;
+# Д, К, Л, М, П, У – 2 очка;
+# Б, Г, Ё, Ь, Я – 3 очка; 
+# Й, Ы – 4 очка;
+# Ж, З, Х, Ц, Ч – 5 очков;
+# Ш, Э, Ю – 8 очков; 
+# Ф, Щ, Ъ – 10 очков.
+# Напишите программу, которая вычисляет стоимость введенного пользователем слова. 
+# Будем считать, что на вход подается только одно слово, которое содержит либо только английские, либо только русские буквы.
+# *Пример:*
+# ноутбук
+# 12
+
+skrabble = {1:'AEIOULNSTRАВЕИНОРСТ', 2:'DGДКЛМПУ', 3:'BCMPБГЁЬЯ', 4:'FHVWYЙЫ', 5:'KЖЗХЦЧ', 8:'JXШЭЮ', 10:'QZФЩЪ'}
+w = str(input('Введите слово: ')).upper()
+print('Колличество очков за слово = ', sum([k for i in w for k, v in skrabble.items() if i in v]))
```

Comment:
Функция `sum` принимает любой итератор, то есть можно передать внутри круглых скобок прямой генератор 
```suggestion
print('Количество очков за слово = ', sum(k for i in w for k, v in skrabble.items() if i in v))
```